scripts/storage-emulator-integration/tests.ts

```diff
@@ -2111,121 +1877,171 @@ describe("Storage emulator", () => {
         });
       });
     });
+  });
 
-    emulatorSpecificDescribe("Non-SDK Endpoints", () => {
-      beforeEach(async () => {
-        await resetStorageEmulator(STORAGE_EMULATOR_HOST);
+  describe("Non-SDK Endpoints", () => {
```

Comment:
These tests will run when pointed to prod servers too now right? Do they all work when pointed to prod?

Reply:
They *should* work when pointed to prod but many are broken. We'll need to do another round of changes to fix the conformance tests but for now I'm re-enabling these when pointing tests to prod servers to make this expectation explicit.

Reply:
Do you mind adding a TODO to indicate that these don't yet work? We can remove it once we get around to fixing them later.

Reply:
done